src/python/tests/flatfiles/config/mlss_conf.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<compara_db division="benchmark">
+
+  <collections>
+
+    <collection name="default">
+      <taxonomic_group taxon_name="Chordata">
+        <ref_for_taxon name="mus_musculus"/>
+        <ref_for_taxon name="sus_scrofa"/>
+      </taxonomic_group>
+      <genome name="drosophila_melanogaster"/>
+      <genome name="caenorhabditis_elegans"/>
+      <genome name="saccharomyces_cerevisiae"/>
+    </collection>
+
+    <collection name="test"/>
+
+  </collections>
+
+</compara_db>
+
```

Comment:
Another idea, It would be nice to add some comments about each XML types... just a brief explanation ?

Reply:
This would also apply to other XML files.

Reply:
Do you mean explanations for `taxonomic_group`, `ref_for_taxon`?